recipe/meta.yaml

```diff
@@ -45,35 +44,37 @@ requirements:
     - python
     - pip
     - cython
-    - arrow-cpp >=10.0.1,<11
-    - pyarrow
-    - numpy
-    - setuptools >=40.6.0
+    - arrow-cpp 10.0.1
+    - pyarrow  10.0.1
+    - numpy  {{ numpy }}
+    - setuptools
```

Comment:
I think the lower bound for `setuptool` can stay, as we discussed such version restriction help reflect what is coming from upstream.

Reply:
I know the linter will complain about this. I think we should also discuss whether to change the linter behavior about non-exact pinning for certain `host` dependencies.

Reply:
Not sure about this. It would have to be exact pinning in host, plus we typically leave setuptools unpinned. It will use the latest version.